.phpstorm.meta.php

```diff
@@ -1,1365 +0,0 @@
-<?php
```

Comment:
If you want to exclude this locally (the way you should), add its path/directory to the _file_ `.git/info/exclude` in the same manner as a `.gitignore` file